core/testing/runner.odin

```diff
@@ -44,7 +45,8 @@ SHARED_RANDOM_SEED    : u64    : #config(ODIN_TEST_RANDOM_SEED, 0)
 LOG_LEVEL             : string : #config(ODIN_TEST_LOG_LEVEL, "info")
 // Show only the most necessary logging information.
 USING_SHORT_LOGS      : bool   : #config(ODIN_TEST_SHORT_LOGS, false)
-
+// Output a report of the tests to the given path.
+JSON_REPORT           : string : #config(ODIN_TEST_JSON_REPORT, "")
```

Comment:
When trying this, my first instinct was to write `-define:ODIN_TEST_JSON_REPORT=true`. I'm trying to think of what might be a less ambiguous config name without being too long. `ODIN_TEST_SAVE_JSON_REPORT_TO`? The open preposition hints at what sort of argument it takes.